test/src/test/kotlin/graphql/nadel/tests/EngineTests.kt

```diff
@@ -180,7 +180,11 @@ private suspend fun execute(
 
                                 if (indexOfCall != null) {
                                     val serviceCall = serviceCalls.removeAt(indexOfCall)
-                                    serviceCall.response
+                                    if (serviceCall.incrementalResponse != null) {
```

Comment:
let's fail the test when both `response` and `incrementalResponse` are defined in the fixture? It never makes sense to define both together

Reply:
👍